vmdb/app/models/ems_refresh/save_inventory_infra.rb

```diff
@@ -298,4 +299,33 @@ def save_storage_files_inventory(storage, hashes)
     deletes = storage.storage_files(true).dup
     save_inventory_multi(:storage_files, storage, hashes, deletes, :name)
   end
+
+  def save_orchestration_stacks_inventory(ems, hashes, target = nil)
```

Comment:
Disregard that comment, I see the refactoring in https://github.com/ManageIQ/manageiq/pull/1899.